plugins/Login/templates/invitation.twig

```diff
@@ -1,48 +1,55 @@
+{% set isCenteredLogo = false %}
+{% set showImageDiv = true %}
+{% set imageToShow = 'plugins/Login/images/invitation-image.png' %}
+{% set colClassOverride = 'col l4 offset-l2 m12' %}
+{% set bodyClass = 'onboarding' %}
 {% extends '@Login/loginLayout.twig' %}
 
 {% block loginContent %}
-    <div class="contentForm invitationForm">
-        <div class="card">
-            <div class="card-content">
-                <div class="card-title">
-                    {{ "Login_InvitationTitle"|translate }}
-                </div>
+    <div class="contentForm invitationForm" id="invitationForm">
+        <h1>{{ 'General_CreatePassword'|translate }}</h1>
+        <p class="description">{{ 'Login_CreatePasswordDescription'|translate }}</p>
 
-                {% if AccessErrorString is defined %}
-                    <div vue-entry="CoreHome.Notification"
-                         noclear="true"
-                         context="error">
-                        <strong>{{ 'General_Error'|translate }}</strong>: {{ AccessErrorString|raw }}<br/>
+        {% if AccessErrorString is defined %}
+            <div vue-entry="CoreHome.Notification"
+                 noclear="true"
+                 context="error">
+                <strong>{{ 'General_Error'|translate }}</strong>: {{ AccessErrorString|raw }}<br/>
+            </div>
+        {% endif %}
+        {% block content %}
+            <form method="post"
+                  action="{{ linkTo({'module': loginPlugin, 'action': 'acceptInvitation', 'token': null}) }}">
+                <input type="hidden" name="token" value="{{ token }}"/>
+                <div class="row">
+                    <div class="col l10 s12 input-field">
+                        <input type="text" name="email" id="email" value="{{ user.email }}" size="20" readonly="readonly"
```

Comment:
This could be tricky if we have emails that are not valid emails e.g. in local installs... what do you think @sgiehl?

```suggestion
                        <input type="email" name="email" id="email" value="{{ user.email }}" size="20" readonly="readonly"
```

Reply:
There is actually no bigger benefit in having the type email here. The field is readonly and the value is provided by us. So if the browser validation wold fail and the user can't submit the form, there would be nothing the user could do.

Reply:
I missed the readonly attribute, so you are (as always 👍) correct.